Makefile

```diff
@@ -2,9 +2,23 @@ ARCH := x86_64
 NAME := cflinuxfs4
 BASE := ubuntu:jammy
 BUILD := $(NAME).$(ARCH)
+COMPAT_BUILD := $(NAME)-compat.$(ARCH)
 
 all: $(BUILD).tar.gz
 
+compat:
+	if [ -z "$(version)" ]; then >&2 echo "version must be set" && exit 1; fi
+	docker build . \
+		--file compat.Dockerfile \
+		--build-arg "base=cloudfoundry/cflinuxfs4:$(version)" \
+		--build-arg packages="`cat "packages/cflinuxfs4-compat" 2>/dev/null`" \
+  	--no-cache "--iidfile=$(COMPAT_BUILD).iid"
+
+	docker run "--cidfile=$(COMPAT_BUILD).cid" `cat "$(COMPAT_BUILD).iid"` dpkg -l | tee "packages-list"
```

Comment:
Nit: can you remove (or not generate packages-list) if not used

Reply:
yes definitely. I was planning to do something with it but its not needed for this iteration so I'll just get rid.

Reply:
fixed, although the dpkg -l remains because we have to run some kind of command or the docker run fails with 
`docker: Error response from daemon: No command specified.`